optaplanner-website-docs/supplemental-ui/partials/head-scripts.hbs

```diff
@@ -0,0 +1,15 @@
+<!-- Adobe Analytics for Red Hat - DPAL (DTM Property Auto-Loader) - part 1/2 -->
+<script id="dpal" src="https://www.redhat.com/dtm.js" type="text/javascript"></script>
+<!-- Google Analytics for kie team: Global site tag (gtag.js) -->
+{{#with site.keys.googleAnalytics}}
+<script async src="https://www.googletagmanager.com/gtag/js?id={{this}}"></script>
+<script>
+    function gtag() {
+        dataLayer.push(arguments)
+    };
+    window.dataLayer=window.dataLayer || [];
+    gtag('js',new Date());
+    gtag('config','{{this}}')
```

Comment:
This script is different than the one in optaplanner-website/templates/base.ftl line 14.

I suspect that's because the one in the optaplanner-docs header went out of sync with the one on our website.
I do believe that the one on our website is the most up to date.

Just checked the diff, it's pretty much the same script, just 2 lines switched.

Reply:
The order of the lines does not matter, as the `window.dataLayer` is initialized before the `gtag()` is called anyway.

But I will change the order to keep the scrips as much similar to each other as possible.